### ISSUE TITLE
[1.11] Configuration File not Generating

### ISSUE THREAD

**Issue by SnowShock35** [progwml6/ironchest#81]
Hey Prog,

Quick (I believe) bug report for you regarding Iron Chests for Minecraft 1.11. Here's the details. . . 

### Important Information

- Minecraft 1.11
- Minecraft Forge 13.19.0.2154
- Iron Chests 7.0.17.799

### Issue Description

The configuration file for ironchests doesn't generate after loading the game. No idea why. Perhaps a change to how configuration files work in 1.11?

**Comment by alexbegt** [comment]
I don't believe ironchests has a config file anymore, as the only two things in the config was to enable or disable cats sitting on the chests and to enable or disable the rendering of items in the crystal chests.
